README.md

```diff
@@ -48,159 +47,125 @@ limitations under the License.
     </a>
 </p>
 
-## Overview
-
-The DeepSparse Engine is a CPU runtime that delivers GPU-class performance by taking advantage of sparsity (read more about sparsification [here](https://docs.neuralmagic.com/main/source/getstarted.html#sparsification)) within neural networks to reduce compute required as well as accelerate memory bound workloads. It is focused on model deployment and scaling machine learning pipelines, fitting seamlessly into your existing deployments as an inference backend.
-
-The [GitHub repository](https://github.com/neuralmagic/deepsparse) includes package APIs along with examples to quickly get started benchmarking and inferencing sparse models.
-
-<img src="https://docs.neuralmagic.com/docs/source/infographics/deepsparse.png" width="960px" />
-
-## Highlights
-
-<p>
-    <a href="https://neuralmagic.com/blog/benchmark-resnet50-with-deepsparse/">
-        <img alt="ResNet-50, b64 - ORT: 296 images/sec vs DeepSparse: 2305 images/sec on 24 cores" src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/resnet-50.png" width="256px" />
-    </a>
-    <a href="https://neuralmagic.com/blog/benchmark-yolov3-on-cpus-with-deepsparse/">
-        <img alt="YOLOv3, b64 - PyTorch: 6.9 images/sec vs. DeepSparse: 46.5 images/sec" src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/yolov3.png" width="256px" />
-    </a>
-</p>
-
-
-## Tutorials
+<h3> Neural Network Inference Engine that Delivers GPU-class Performance for Sparsified Models on CPUs </h3>
```

Comment:
Refer to my earlier comment about reverting back to the original location of this tag line.